cmd/observe/observe.go

```diff
@@ -265,13 +267,13 @@ func handleArgs(
 	if ignoreStderr {
 		opts = append(opts, hubprinter.IgnoreStderr())
 	}
-	printer = hubprinter.New(opts...)
-
 	if numeric {
-		format.EnableIPTranslation = false
-		format.EnablePortTranslation = false
+		enableIPTranslation = false
+		enablePortTranslation = false
 	}
-
+	opts = append(opts, hubprinter.SetIPTranslation(enableIPTranslation))
+	opts = append(opts, hubprinter.SetPortTranslation(enablePortTranslation))
```

Comment:
only makes sense to append these options if the values are true, really

Reply:
will fix